src/Php/PhpVersion.php

```diff
@@ -66,7 +66,7 @@ public function deprecatesRequiredParameterAfterOptionalUnionOrMixed(): bool
 		return $this->versionId >= 80300;
 	}
 
-	public function supportsLessOverridenParametersWithVariadic(): bool
+	public function supportsLessOverriddenParametersWithVariadic(): bool
```

Comment:
the PhpVersion class is declared `@api` therefore you cannot change this public method name here (BC break)

Reply:
I've fixes a method name in `PhpVersion` although it is marked `@api`.
Seems to be unused: https://github.com/search?q=supportsLessOverridenParametersWithVariadic+NOT+is%3Afork&type=code

Reply:
Should I revert it?

Reply:
this method might be used in 3rd party extensions (thats what the `@api` is meant for)

Reply:
Reverted 🍏